[BUG]: Warnings when compiling version 4.0.5 on Alpine

**Describe the bug**

Get warnings when compiling version 4.0.5 on Alpine:
```
/tmp/pear/temp/phalcon/phalcon.zep.c: In function 'zephir_mt_rand':
/tmp/pear/temp/phalcon/phalcon.zep.c:12795:45: warning: format '%lld' expects argument of type 'long long int', but argument 4 has type 'zend_long' {aka 'long int'} [-Wformat=]
12795 |   php_error_docref(NULL, E_WARNING, "max(%lld) is smaller than min(%lld)", max, min);
      |                                          ~~~^                              ~~~
      |                                             |                              |
      |                                             long long int                  zend_long {aka long int}
      |                                          %ld
/tmp/pear/temp/phalcon/phalcon.zep.c:12795:71: warning: format '%lld' expects argument of type 'long long int', but argument 5 has type 'zend_long' {aka 'long int'} [-Wformat=]
12795 |   php_error_docref(NULL, E_WARNING, "max(%lld) is smaller than min(%lld)", max, min);
      |                                                                    ~~~^         ~~~
      |                                                                       |         |
      |                                                                       |         zend_long {aka long int}
      |                                                                       long long int
      |                                                                    %ld
```

But everything seems to work:
```
# php -m | grep phalcon
phalcon
# php -r "echo \Phalcon\Version::get() . PHP_EOL;"
4.0.5
```

**To Reproduce**

Dockerfile:
```
FROM php:7.4-cli-alpine
ARG PHALCON_VERSION=4.0.5
ARG DEPS_BUILD="autoconf build-base pcre-dev"
RUN set -ex \
    && apk add --no-cache --virtual .build-deps $DEPS_BUILD \
    && curl -sSLo /tmp/phalcon.tar.gz https://codeload.github.com/phalcon/cphalcon/tar.gz/v$PHALCON_VERSION \
    && cd /tmp/ && tar xzf phalcon.tar.gz \
    && cd cphalcon-$PHALCON_VERSION/build && sh install \
    && docker-php-ext-enable phalcon --ini-name docker-php-ext-phalcon.ini \
    && apk del --no-cache .build-deps \
    && rm -rf /tmp/cphalcon* /tmp/phalcon*
```

The same when installing from PECL.

I have same problem on PHP 7.4.4 on Ubuntu 19.10

Installation with pecl shows the below error on terminal and after 15 minutes it succeed!

```
/tmp/pear/temp/phalcon/phalcon.zep.c: In function ‘zephir_mt_rand’:
/tmp/pear/temp/phalcon/phalcon.zep.c:12795:45: warning: format ‘%lld’ expects argument of type ‘long long int’, but argument 4 has type ‘zend_long’ {aka ‘long int’} [-Wformat=]
12795 |   php_error_docref(NULL, E_WARNING, "max(%lld) is smaller than min(%lld)", max, min);
      |                                          ~~~^                              ~~~
      |                                             |                              |
      |                                             long long int                  zend_long {aka long int}
      |                                          %ld
/tmp/pear/temp/phalcon/phalcon.zep.c:12795:71: warning: format ‘%lld’ expects argument of type ‘long long int’, but argument 5 has type ‘zend_long’ {aka ‘long int’} [-Wformat=]
12795 |   php_error_docref(NULL, E_WARNING, "max(%lld) is smaller than min(%lld)", max, min);
      |                                                                    ~~~^         ~~~
      |                                                                       |         |
      |                                                                       |         zend_long {aka long int}
      |                                                                       long long int
      |                                                                    %ld
```

--- a/ext/kernel/math.c
+++ b/ext/kernel/math.c
@@ -195,7 +195,7 @@ zephir_mt_rand(zend_long min, zend_long max)
 	zend_long number;
 
 	if (max < min) {
-		php_error_docref(NULL, E_WARNING, "max(%lld) is smaller than min(%lld)", max, min);
+		php_error_docref(NULL, E_WARNING, "max(%ld) is smaller than min(%ld)", max, min);
 		return 0;
 	}
 

--- a/kernels/ZendEngine3/math.c
+++ b/kernels/ZendEngine3/math.c
@@ -195,7 +195,7 @@ zephir_mt_rand(zend_long min, zend_long max)
 	zend_long number;
 
 	if (max < min) {
-		php_error_docref(NULL, E_WARNING, "max(%lld) is smaller than min(%lld)", max, min);
+		php_error_docref(NULL, E_WARNING, "max(%ld) is smaller than min(%ld)", max, min);
 		return 0;
 	}
 

